src/frontend/Komponenter/Uttrekk/UttrekkArbeidssøker.tsx

```diff
@@ -0,0 +1,229 @@
+    årMåned: string;
+    side: number;
+    antallTotalt: number;
+    antallKontrollert: number;
+    arbeidssøkere: UttrekkArbeidsssøker[];
+}
+
+interface UttrekkArbeidsssøker {
+    id: string;
+    fagsakId: string;
+    behandlingIdForVedtak: string;
+    kontrollert: boolean;
+    kontrollertTid?: string;
+    kontrollertAv?: string;
+}
+
+const URL_ARBEIDSSØKER = '/familie-ef-sak/api/uttrekk/arbeidssoker';
+
+const settArbeidssøkereTilKontrollert = (
+    prevState: RessursSuksess<UttrekkArbeidssøkere>,
+    id: string,
+    kontrollert: boolean
+) => {
+    return {
+        ...prevState,
+        data: {
+            ...prevState.data,
+            arbeidssøkere: prevState.data.arbeidssøkere.map((arbeidssøker) =>
+                arbeidssøker.id === id ? { ...arbeidssøker, kontrollert } : arbeidssøker
+            ),
+        },
+    };
+};
+
+const ANTALL_PER_SIDE = 20;
+const QUERY_PARAM_SIDE = 'side';
+const QUERY_PARAM_KONTROLLERTE = 'kontrollerte';
+
+const UttrekkArbeidssøker: React.FC = () => {
+    const query = useQueryParams();
+    const history = useHistory();
+
+    const side = query.get(QUERY_PARAM_SIDE) || 1;
+    const visKontrollerte = query.get(QUERY_PARAM_KONTROLLERTE) === 'true';
+
+    const [arbeidssøkere, settArbeidssøkere] = useState<Ressurs<UttrekkArbeidssøkere>>(
+        byggTomRessurs()
+    );
+    const [feilmelding, settFeilmelding] = useState<string>();
+    const { axiosRequest, gåTilUrl } = useApp();
+
+    const hentUttrekk = useCallback(
+        (side, visKontrollerte: boolean) => {
+            return axiosRequest<UttrekkArbeidssøkere, null>({
+                method: 'GET',
+                url: `${URL_ARBEIDSSØKER}?side=${side}${
+                    visKontrollerte ? '&visKontrollerte=true' : ''
+                }`,
+            }).then((respons: RessursSuksess<UttrekkArbeidssøkere> | RessursFeilet) => {
+                if (respons.status === RessursStatus.SUKSESS) {
+                    settArbeidssøkere(respons);
+                } else {
+                    settFeilmelding(respons.frontendFeilmelding);
+                }
+            });
+        },
+        [axiosRequest]
+    );
+
+    const settKontrollert = useCallback(
+        (id: string, kontrollert: boolean): void => {
+            const kontrollertQuery = kontrollert ? '?kontrollert=false' : '';
+            axiosRequest<UttrekkArbeidssøkere, null>({
+                method: 'POST',
+                url: `${URL_ARBEIDSSØKER}/${id}/kontrollert${kontrollertQuery}`,
+            }).then((respons: RessursSuksess<UttrekkArbeidssøkere> | RessursFeilet) => {
+                settFeilmelding(undefined);
+                if (respons.status === RessursStatus.SUKSESS) {
+                    settArbeidssøkere((prevState) => {
+                        if (prevState.status !== RessursStatus.SUKSESS) {
+                            throw Error('Kan ikke oppdatere prevstate som ikke har status suksess');
+                        }
```

Comment:
Hvordan håndterer vi dette ut mot sluttbruker egentlig? Ser for meg at appen bare "krasjer" ...?

En form for feilmelding med "trykk her for å oppdatere" høres litt mer hyggelig ut, kanskje?